apps/src/client.rs

```diff
@@ -432,11 +461,17 @@ pub fn connect(
                 quiche::PathEvent::New(..) => unreachable!(),
 
                 quiche::PathEvent::Validated(local_addr, peer_addr) => {
+                    trace!("PathEvent::Validated");
                     info!(
                         "Path ({}, {}) is now validated",
                         local_addr, peer_addr
                     );
-                    conn.migrate(local_addr, peer_addr).unwrap();
+                    if migrated {
```

Comment:
Would adding logs for these option help?

Reply:
added 👍